### ISSUE TITLE
Create a Streamlit based app to run a generic MDF model

### ISSUE THREAD

**Issue by pgleeson** [ModECI/MDF#536]
Being worked on by @rimjhimittal as part of Outreachy

@rimjhimittal I've just thought, in addition to tabs below the list of parameters for 1) traces of outputs of the ran model 2) generated graph of the model, it would be good to have another tab 3) actual MDF model code in JSON or YAML

**Comment by rimjhimittal** [comment]
That's a great idea! i'll add it.

**Comment by rimjhimittal** [comment]
@pgleeson can our project be like this in future that, for example, i upload a tensorflow or pytorch model on our streamlit application, and using pytorch_to_mdf() function, it creates visualisations and input fields and everything even for that?

**Comment by pgleeson** [comment]
Certainly! Not every Pytorch/TF model will be approriate, but this can be checked and the user informed. Ideally it could be a one stop shop for loading (m)any of the formats in the image below, e.g. PyTorch./NeuroML -> mapping to MDF -> listing of parameters, visualisation of structure -> user edits values, changes inputs -> maps back to original or other format for optimal execution -> visualise results. 

![image](https://github.com/ModECI/MDF/assets/1556687/33aebb44-2483-45c5-97ae-3b2f6df68396)

**Comment by pgleeson** [comment]
Completed and running here:  https://huggingface.co/spaces/ModECI/MDF-UI
